### ISSUE TITLE
We should really separate contacts and messages, these are two different components...

### ISSUE THREAD

**Issue by victorjzsun** [Yuxuan-Lin/VioLegacy#32]
We should really separate contacts and messages, these are two different components...

_Originally posted by @victorjzsun in https://github.com/Yuxuan-Lin/VioLegacy/pull/31#r505799374_

**Comment by Yuxuan-Lin** [comment]
You are right. We found our old code to be kind of retarded. We are looking at your new pull request of reshaping the Message.
